codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/sharedstream/person.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.sharedeventstream.model;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.core.SdkField;
+import software.amazon.awssdk.core.SdkPojo;
+import software.amazon.awssdk.core.protocol.MarshallLocation;
+import software.amazon.awssdk.core.protocol.MarshallingType;
+import software.amazon.awssdk.core.traits.LocationTrait;
+import software.amazon.awssdk.utils.ToString;
+import software.amazon.awssdk.utils.builder.CopyableBuilder;
+import software.amazon.awssdk.utils.builder.ToCopyableBuilder;
+
+/**
+ */
+@Generated("software.amazon.awssdk:codegen")
+public final class Person implements SdkPojo, Serializable, ToCopyableBuilder<Person.Builder, Person>, EventStream {
+  private static final SdkField<String> NAME_FIELD = SdkField.<String>builder(MarshallingType.STRING)
+  .getter(getter(Person::name))
+  .setter(setter(Builder::name))
+  .traits(LocationTrait.builder()
+  .location(MarshallLocation.PAYLOAD)
+  .locationName("Name")
+  .build()).build();
+
+  private static final SdkField<Instant> BIRTHDAY_FIELD = SdkField.<Instant>builder(MarshallingType.INSTANT)
+  .getter(getter(Person::birthday))
+  .setter(setter(Builder::birthday))
+  .traits(LocationTrait.builder()
+  .location(MarshallLocation.PAYLOAD)
+  .locationName("Birthday")
+  .build()).build();
+
+  private static final List<SdkField<?>> SDK_FIELDS = Collections.unmodifiableList(Arrays.asList(NAME_FIELD,BIRTHDAY_FIELD));
+
+  private static final long serialVersionUID = 1L;
+
+  private final String name;
+
+  private final Instant birthday;
+
+  private Person(BuilderImpl builder) {
+    this.name = builder.name;
+    this.birthday = builder.birthday;
+  }
+
+  /**
+   * Returns the value of the Name property for this object.
+   * @return The value of the Name property for this object. */
+  public String name() {
+    return name;}
```

Comment:
Formatting seems off. Did the generator do this? Does this compile?

Reply:
Yeah, I just copy and pasted the output from the generator and reviewed it to make sure it generated the methods I wanted.  IDEA only complains about missing types when I move it into a source root.

Reply:
Can we run a quick formatter on it? The comparison doesn't care about whitespace, I don't think.

Reply:
Ran IDEA's formatter on all of the newly added generated classes.

Reply:
And broke the tests.  Don't merge this yet.

Reply:
IDEA changed

```
/*
 */
```
to

```
/*
 *
 */
```

which, of course, is not whitespace-only.  I pushed an update, should be good as soon as the automated tests don't call me a liar.